src/packagedcode/models.py

```diff
@@ -1633,10 +1639,11 @@ def to_package_data(self):
         mapping.pop('package_uid', None)
         mapping.pop('datafile_paths', None)
         mapping.pop('datasource_ids', None)
+        mapping.pop('documentation_url', None)
         return PackageData.from_dict(mapping)
 
     @classmethod
-    def from_package_data(cls, package_data, datafile_path=None, package_only=False):
+    def from_package_data(cls, package_data, datafile_path=None, package_only=False , documentation_url=None):
```

Comment:
These changes are not correct, please remove. You have to make the changes in the individual datafile handlers, this is a generic function used to create top-level Package objects from PackageData. All common attributes are inherited automatically